tests/measurement.rs

```diff
@@ -363,7 +363,7 @@ mod snp_tests {
     }
 }
 
-#[cfg(all(target_os = "linux", feature = "sev"))]
+#[cfg(all(target_os = "linux", feature = "sev", feature = "openssl"))]
```

Comment:
Did you only add support for SNP?

Reply:
Yes I guess the commit title says otherwise, I can add that too